SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -541,27 +591,96 @@ private static int CreateSimDeployments(string[] args, bool useChinaPlatform)
 
             if (!String.IsNullOrEmpty(clusterCode))
             {
-                simDeploymentConfig.ClusterCode = clusterCode;
+                simDeployment.ClusterCode = clusterCode;
             }
             else
             {
-                simDeploymentConfig.RegionCode = regionCode;
+                simDeployment.RegionCode = regionCode;
             }
 
-            simDeploymentConfig.Tag.Add(DEPLOYMENT_LAUNCHED_BY_LAUNCHER_TAG);
-            simDeploymentConfig.Tag.Add(SIM_PLAYER_DEPLOYMENT_TAG);
+            simDeployment.Tag.Add(DEPLOYMENT_LAUNCHED_BY_LAUNCHER_TAG);
+            simDeployment.Tag.Add(SIM_PLAYER_DEPLOYMENT_TAG);
 
             Console.WriteLine(
-                $"Creating the simulated player deployment {simDeploymentName} in project {projectName} with {simNumPlayers} simulated players. Link: https://console.improbable.io/projects/{projectName}/deployments/{simDeploymentName}/overview");
+                $"Creating the simulated player deployment {simDeploymentName} in project {projectName} with {numSimPlayers} simulated players. Link: https://console.improbable.io/projects/{projectName}/deployments/{simDeploymentName}/overview");
 
             var simDeploymentCreateOp = deploymentServiceClient.CreateDeployment(new CreateDeploymentRequest
             {
-                Deployment = simDeploymentConfig
+                Deployment = simDeployment
             });
 
             return simDeploymentCreateOp;
         }
 
+        private static List<Operation<Deployment, CreateDeploymentMetadata>> CreateSimPlayerDeploymentsAsync(DeploymentServiceClient deploymentServiceClient,
+            string projectName, string assemblyName, string runtimeVersion, string mainDeploymentName, string simDeploymentBaseName,
+            string simDeploymentJsonPath, string simDeploymentSnapshotPath, string regionCode, string clusterCode, int numSimPlayers, int maxPlayersPerDeployment, bool useChinaPlatform)
+        {
+            var operations = new List<Operation<Deployment, CreateDeploymentMetadata>>();
+
+            var numSimDeployments = (int)Math.Ceiling(numSimPlayers / (double)maxPlayersPerDeployment);
+
+            var longestName = GetSimDeploymentName(simDeploymentBaseName, numSimDeployments);
+            if (longestName.Length > 32)
+            {
+                Console.WriteLine($"The deployment name may not exceed 32 characters. '{longestName}' would have {longestName.Length}.");
+                return operations;
+            }
+
+            for (var simPlayerDeploymentId = 1; simPlayerDeploymentId <= numSimDeployments; ++simPlayerDeploymentId)
+            {
+                var simDeploymentName = simDeploymentBaseName;
+                if (simPlayerDeploymentId != 1)
+                {
+                    simDeploymentName += ("_" + simPlayerDeploymentId);
```

Comment:
use `GetSimDeploymentName`?

Reply:
Yes. I added the function but then forgot to replace the old code. Fixed 👍